modules/bitgo/src/v2/coins/tdot.ts

```diff
@@ -18,4 +26,8 @@ export class Tdot extends Dot {
   getFullName(): string {
     return 'Testnet Polkadot';
   }
+
+  getBaseFactor(): number {
+    return Math.pow(10, this._staticsCoin.decimalPlaces);
```

Comment:
Is this suppose to be 12 like in static?
https://github.com/BitGo/BitGoJS/blob/master/modules/statics/src/coins.ts#L78